### PR TITLE
Add new codec HEVC with ID=5 for HDBits

### PR DESCRIPTION
**Pull request by Wyall**
According to LSL1337 on Radarr Discord there's a new codec option HEVC with ID=5

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs [hunk 1]
```diff
@@ -58,7 +58,8 @@ public enum HdBitsCodec
         H264 = 1,
         Mpeg2 = 2,
         Vc1 = 3,
-        Xvid = 4
+        Xvid = 4.
+        HEVC = 5
```
Comment (markus101) [inline src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs]: Should be `Hevc` for code styling consistency.

File: src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs [hunk 2]
```diff
@@ -58,7 +58,8 @@ public enum HdBitsCodec
         H264 = 1,
         Mpeg2 = 2,
         Vc1 = 3,
-        Xvid = 4
+        Xvid = 4.
```
Comment (thezoggy) [inline src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs]: shouldnt that be a trailing coma not a period. also squash and rebase your commits to keep history clean

Comment (markus101) [reply src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs]: Yes, should be a `,`.

During a review squashing and rebasing should be left to the very end so individual commits can be compared. (We can squash and merge right from GitHub now).

Comment (Wyall) [reply src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs]: Did I do that wrong (squashing and rebasing)? I'm sorry, github can be complicated if not used frequently

**Pull request merged**